src/main/java/core/basesyntax/bd/Storage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.bd;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitTransaction> fruitTransaction = new ArrayList<>();
```

Comment:
Storage should keep Fruits, not FruitTransactions
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```